hoot-core-test/src/test/cpp/hoot/core/TestUtils.cpp

```diff
@@ -143,9 +143,7 @@ void TestUtils::dumpString(const string& str)
     {
       cout << endl << "  ";
     }
-    //printf("%3d%c", (unsigned char)str.at(i),
-    //  str.at(i) >= 32 && str.at(i) <= 125 ? (char)(str.at(i)) : '#');
-    printf("%3d", (unsigned char)str.at(i));
+    printf("%3u", (unsigned char)str.at(i));
```

Comment:
Caught a couple of warnings about using the correct type `u` for unsigned rather than `d` for signed.  Not a big deal but not a big change.